Recommend using Datadog CLI to enable serverless monitoring (#11376)
DataDog/documentation: content/en/serverless/guide/datadog_forwarder_node.md

## Configuration

{{< tabs >}}
{{% tab "Datadog CLI" %}}

> Datadog CLI modifies existing Lambda functions' configuration to enable instrumentation *without requiring a new deployment*. It is the *quickest* way to get started with Datadog serverless monitoring.

Reviewer: ```suggestion
The Datadog CLI modifies existing Lambda functions' configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
```

> You can also add the command to your CI/CD pipelines to enable instrumentation for *all* your serverless applications. Run the command *after* your normal serverless application deployment, so the changes made by the Datadog CLI command don't get overridden.

Reviewer: ```suggestion
You can also add the command to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command *after* your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```
